### PR TITLE
Injimob 1709 kotlin vc verifier

### PR DESCRIPTION
**Pull request by sree96**
This PR contains the changes that are required for support in Android

### CODE REVIEW

**Review by Alka1703 — APPROVED** [review]
LGTM

File: vc-verifier/kotlin/vcverifier/src/main/java/io/mosip/vercred/vcverifier/CredentialsVerifier.kt [hunk 1]
```diff
@@ -36,74 +38,79 @@ import java.util.Objects
...
-            Log.i(tag,"public key download completed.")
-            val strReader = StringReader(publicKeyPem)
-            val pemReader = PemReader(strReader)
-            val pemObject: PemObject = pemReader.readPemObject()
-            val pubKeyBytes: ByteArray = pemObject.content
-            val pubKeySpec = X509EncodedKeySpec(pubKeyBytes)
-            val keyFactory = KeyFactory.getInstance("RSA")
-            return keyFactory.generatePublic(pubKeySpec)
+        return try {
+            val okHttpClient = OkHttpClient.Builder().build().newBuilder().build()
```
Comment (vijay151096) [inline vc-verifier/kotlin/vcverifier/src/main/java/io/mosip/vercred/vcverifier/CredentialsVerifier.kt:84]: @sree96 , last time when we tried to integrate a library with Inji, OkHttpClient was not compatible android JVM. can we try to integrate with inji app and verify once. or we can use other HTTP Clients

Comment (sree96) [reply vc-verifier/kotlin/vcverifier/src/main/java/io/mosip/vercred/vcverifier/CredentialsVerifier.kt:84]: We have tried this @vijay151096 , The older implementation which was java.net.http.HttpClient was the one incompatible with Android. These changes with OkHttpClient we have already tested on Android and is working fine. 

cc: @Alka1703

Comment (vijay151096) [reply vc-verifier/kotlin/vcverifier/src/main/java/io/mosip/vercred/vcverifier/CredentialsVerifier.kt:84]: Okay Fine then, all good here.

**Review by vijay151096 — APPROVED** [review]
lgtm

**Review by swatigoel — APPROVED** [review]

**Pull request merged**